refactored variables containing master

gooney152 opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide/first-contribution.md#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
5. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-->

#### What type of PR is this?
/kind cleanup
/wg-naming

<!--
Add one of the following kinds:
/kind bug
/kind cleanup
/kind documentation
/kind feature
/kind design

Optionally add one or more of the following kinds if applicable:
/kind api-change
/kind deprecation
/kind failing-test
/kind flake
/kind regression
-->

#### What this PR does / why we need it:
updating the new name references master-to-control-plane

#### Which issue(s) this PR fixes:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #

#### Special notes for your reviewer:

#### Does this PR introduce a user-facing change?
NONE
<!--
If no, just write "NONE" in the release-note block below.
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".

For more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md
-->
```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

<!--
This section can be blank if this pull request does not require a release note.

When adding links which point to resources within git repositories, like
KEPs or supporting documentation, please reference a specific commit and avoid
linking directly to the master branch. This ensures that links reference a
specific point in time, rather than a document that may change over time.

See here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files

Please use the following format for linking documentation:
- [KEP]: <link>
- [Usage]: <link>
- [Other doc]: <link>
-->
```docs

```

marosset committed:
Updating OWNERS_ALIASES for SIG-Windows

cpanato committed:
device-plugins: replace gcr.io/gke-release to use the community registry

Signed-off-by: Carlos Panato <ctadeu@gmail.com>

odinuge committed:
Ensure node e2e apiserver and test suite can open enough files

The apiserver and test suite in node e2e runs under the sshd daemon
that can limit the amount of files it can open. Set a higher limit
to address the issues.

Signed-off-by: Odin Ugedal <odin@uged.al>

ffromani committed:
e2e: increase readiness gate timeout

We're trying to fix https://github.com/kubernetes/kubernetes/issues/75355
sicne long time, and we believe the current timeout could
actually be too low (despite being "forever", which is 30s).

To validate this theory, we set the timeout to one full minute.

Also, make the logging more verbose to make the troubleshooting easier.

Signed-off-by: Francesco Romani <fromani@redhat.com>

wzshiming committed:
Fix dbus config path for GracefulNodeShutdown e2e

gooney152 committed:
refactored variables containing master

gooney152 committed:
refactored variables containing master

gooney152 commented:
/assign @oomichi

gooney152 commented:
/assign @pohly

gooney152 committed:
Merge branch 'kubernetes:master' into only-refactor

gooney152 commented:
/assign @wangyysde

gooney152 commented:
/assign @caesarxuchao

BenTheElder commented:
/ok-to-test

Unfortunately the project does not allow merge commits in PRs, so this PR cannot merge until that is fixed.
https://www.kubernetes.dev/docs/guide/github-workflow/ might be helpful

BenTheElder commented on test/e2e/framework/resource_usage_gatherer.go:
```diff
@@ -603,21 +603,21 @@ type kubemarkResourceUsage struct {
+	sshResult, err := getControlPlaneUsageByPrefix("kube")
 	if err != nil {
-		Logf("Error when trying to SSH to master machine. Skipping probe. %v", err)
+		Logf("Error when trying to SSH to controlplane machine. Skipping probe. %v", err)
```
tiny non-blocking nit: when it's not a variable name we should leave a space between the words
```suggestion
		Logf("Error when trying to SSH to control plane machine. Skipping probe. %v", err)
```

BenTheElder reviewed (commented):
/approve

gooney152 committed:
Update test/e2e/framework/resource_usage_gatherer.go

Co-authored-by: Benjamin Elder <bentheelder@google.com>

gooney152 committed:
Update test/e2e/framework/resource_usage_gatherer.go

Co-authored-by: Benjamin Elder <bentheelder@google.com>

gooney152 commented:
/retest

gooney152 commented:
@BenTheElder

gooney152 committed:
Merge branch 'kubernetes:master' into only-refactor

BenTheElder commented:
https://github.com/kubernetes/kubernetes/pull/103569/commits/16b924e9d6fe99ad4d9669201d41f12fe7e2bd48 and https://github.com/kubernetes/kubernetes/pull/103569/commits/3f5c70a930b6577e11c2c5c21bfe00837a346955 prevent this from being mergeable (see: https://github.com/kubernetes/kubernetes/pull/103569#issuecomment-876646290)

Otherwise these changes look good, I've already approved but I may be out the rest of this and next week so probably someone else will need to `/lgtm` (any member of the project) once the robot is no longer blocking merge due to the merge commits.

Thanks for the PR!

dims committed:
update golangci-lint to newer version

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

dims committed:
fix deadcode issues

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

dims committed:
fix ineffassign and varcheck

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

dims committed:
enable verify-golangci-lint.sh

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

oomichi commented:
/release-note-none

robscott committed:
Revert granting EndpointSlice write access to edit role

This reverts part of the change introduced by #101203.

caesarxuchao commented:
/lgtm
/retest

Commit:
CHANGELOG: Update directory for v1.22.0-beta.2 release

gooney152 committed:
Merge branch 'kubernetes:master' into only-refactor

alculquicondor committed:
Remove E2E test for NodePreferAvoidPods scheduling Score

The feature is now disabled by default. The annotation never graduated from alpha. The same behavior can be achieved with PreferNoSchedule taints.

Commit:
CHANGELOG: Update directory for v1.19.13 release

Commit:
CHANGELOG: Update directory for v1.20.9 release

Commit:
CHANGELOG: Update directory for v1.21.3 release

gooney152 committed:
Merge branch 'kubernetes:master' into only-refactor

gooney152 committed:
Merge branch 'only-refactor' of github.com:gooney152/kubernetes into only-refactor
